src/App.css

```diff
@@ -1,3 +1,33 @@
 main {
   margin-left: 4rem;
+  color: #fff;
```

Comment:
Maybe use CSS variables for setting color instead. 

--color: #fff;
color: var(--color)